src/view/screens/ProfileList.tsx

```diff
@@ -142,12 +148,13 @@ function ProfileListScreenLoaded({
   const {openModal} = useModalControls()
   const isCurateList = list.purpose === 'app.bsky.graph.defs#curatelist'
   const isScreenFocused = useIsFocused()
+  const isHidden = list.labels?.findIndex(l => l.val === '!hide') !== -1
```

Comment:
In the last PR I used the mod SDK and ScreenHider to do this. I like the new screen better, but we may want to borrow the mod SDK handling in case we want to respect other filter labels here. We can suss it out in the morning 👍

Reply:
Yeah I think it'd be nice to keep this sorta functionality around ![CleanShot 2024-08-20 at 07 59 37@2x](https://github.com/user-attachments/assets/b2ab7c3e-ebe3-4043-93ba-dfb2bedc9ff9)

Reply:
Maybe the move is allow an additional component prop on ScreenHider and improve the look of that component if we want. There are a possibility of 2 CTAs right now, so I think instead of adding another button, maybe it'd be nice to allow for a lil alert section above the buttons that explains an additional action that can be taken. Can discuss this morning!

Reply:
Yep added this now.